packages/http-client-python/package.json

```diff
@@ -30,12 +30,12 @@
   },
   "scripts": {
     "clean": "rimraf ./dist ./temp ./emitter/temp ./generator/test/azure/generated ./generator/test/unbranded/generated ./venv",
-    "build": "tsc -p ./emitter/tsconfig.build.json",
+    "build": "tsc -p ./emitter/tsconfig.build.json && tsx ./eng/scripts/setup/run-python3.ts ./eng/scripts/setup/build_pygen_wheel.py && rimraf ./venv_build_wheel",
```

Comment:
nit: i would follow laurent's comments and clean up the package.json scripts as well in this pr